common/src/main/java/com/microsoft/identity/common/internal/authorities/AzureActiveDirectoryB2CAuthority.java

```diff
@@ -93,4 +93,14 @@ public OAuth2Strategy createOAuth2Strategy(@NonNull final OAuth2StrategyParamete
         MicrosoftStsOAuth2Configuration config = createOAuth2Configuration();
         return new MicrosoftStsOAuth2Strategy(config, parameters);
     }
+
+    public String getB2CPolicyName(){
```

Comment:
needs javadoc

Reply:
also let's add some unit tests for this method specifically

Reply:
Noob question, where do I add the unit test?

Reply:
you can add it under `common/common/src/test/java/unit`

Reply:
I am not very confident of my unit test, lets have conversation to update as needed